protected/humhub/components/Theme.php

```diff
@@ -52,9 +52,9 @@ public function init()
             $this->setBasePath('@webroot/themes/' . $this->name);
         }
 
-        $this->pathMap = [
+        $this->pathMap = array_merge($this->pathMap, [
```

Comment:
Build fails because $this->pathMap could be (and is by default) null. Please wrap a if (is_array($this->pathMap)) around array_merge.